src/main/java/school/hei/patrimoine/visualisation/swing/ihm/MainIHM.java

```diff
@@ -67,6 +67,22 @@ private void configureFrame() {
     setLocationRelativeTo(null);
   }
 
+  private JPanel getFluxJournaliersIHM() {
+    var fluxJournaliersBox = new BoxLayout(fluxJournaliersIHM, BoxLayout.X_AXIS);
+    fluxJournaliersIHM.setLayout(fluxJournaliersBox);
+    fluxJournaliersIHM.setBorder(BorderFactory.createEmptyBorder(10, 5, 0, 5));
+
+    Dimension fixedWidth =
+        new Dimension(
+            fluxImpossiblesIHM.getPreferredSize().width,
+            fluxJournaliersIHM.getPreferredSize().height);
```

Comment:
if possible, use the same variable for fluxImpossiblesIHM and fluxJournaliersIHM preferredSize and use the variable